Foundation/UserDefaults.swift

```diff
@@ -299,31 +288,105 @@ open class UserDefaults: NSObject {
         CFPreferencesRemoveSuitePreferencesFromApp(kCFPreferencesCurrentApplication, suiteName._cfObject)
     }
     
-    open func dictionaryRepresentation() -> [String : Any] {
-        guard let aPref = CFPreferencesCopyMultiple(nil, kCFPreferencesCurrentApplication, kCFPreferencesCurrentUser, kCFPreferencesCurrentHost),
-            let bPref = (aPref._swiftObject) as? [NSString: Any] else {
-                return registeredDefaults
+    open func dictionaryRepresentation() -> [String: Any] {
+        return _dictionaryRepresentation(searchingOutsideOfSuite: true)
+    }
+    
+    private func _dictionaryRepresentation(searchingOutsideOfSuite: Bool) -> [String: Any] {
```

Comment:
searchingOutsideOfSuite seems like a slightly odd name for this. “Suite” is a very fuzzy term in NSUD, but to the extent that it means anything I tend to think of it as referring to a search list, which would generally include registered prefs, etc...

I’m guessing the sense you mean here is “outside of the set of domains matching this identifier”, which is a subset of the search list for that identifier?

Reply:
Yup. What about `_dictionaryRepresentation(includingOnlySuiteNameDomains:)`?

Reply:
We chatted privately and agreed on `_dictionaryRepresentation(includingVolatileDomains:)`.